ArduCopter/mode.h

```diff
@@ -198,12 +199,14 @@ class Mode {
 
     // method shared by both Guided and Auto for takeoff.  This is
     // waypoint navigation but the user can control the yaw.
-    void auto_takeoff_run();
-    void auto_takeoff_set_start_alt(void);
+    bool auto_takeoff_run();
+    void auto_takeoff_set_start_and_final_alt(float complete_alt);
```

Comment:
let's change the name to of this argument to "final_alt" and/or "final_alt_cm" and let's add a comment to clarify the frame.